apis/tags/v1alpha1/tagbinding_types.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"github.com/GoogleCloudPlatform/k8s-config-connector/apis/tags/v1beta1"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/apis/k8s/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var TagsLocationTagBindingGVK = GroupVersion.WithKind("TagsLocationTagBinding")
+
+// TagsLocationTagBindingSpec defines the desired state of TagsLocationTagBinding
+// +kcc:spec:proto=google.cloud.resourcemanager.v3.TagBinding
+type TagsLocationTagBindingSpec struct {
+	// +kcc:ref=Project
+	ParentRef *v1beta1.TagsTagBindingParentRef `json:"parentRef"`
+
+	// The location for the resource being tagged.
+	// +required
+	Location string `json:"location"`
```

Comment:
Should this be a *string ? I realize its required but seems like switching to pointer allows for things like the field becoming optional later. Might be easier to just be consistent on this.